README.md

```diff
@@ -37,8 +53,11 @@ If you have one of these cars, please stop by the [comma Discord](https://discor
   * Uses TSS 3.0 but does not appear to have ECU Security Key or SecOC steps when replacing the forward camera. It's unknown whether it has TSK, and if yes in what form. Maybe they just don't do the pairing thing but hardcode a key. No one knows.
 * 2021+ Yaris Cross Hybrid (EUDM/JDM/MXDM)
   * Brute force script may work.
-* 2021+ Yaris GR (EUDM/JDM/MXDM)
+* 2021+ GR Yaris (EUDM/JDM/MXDM)
```

Comment:
I only mention that because we have this.

```
* 2021+ GR Yaris (EUDM/JDM/MXDM)
  * Brute force script may work.
* 2023+ GR Yaris
  * TSS 2.0
  * Seems externally similar to Yaris Hybrid?
```

This could make better sense.
```
* 2021-2022 GR Yaris (EUDM/JDM/MXDM)
  * Brute force script may work.
* 2023+ GR Yaris, All origins.
  * TSS 2.0
  * Seems externally similar to Yaris Hybrid?
```